requirements.txt

```diff
@@ -15,5 +15,6 @@
   pytest-rerunfailures
   pytest-xdist
   selenium==4.1.0
+  tblib
```

Comment:
Do we need tblib installed in the host?

Reply:
Yeah it's needed in the host to unpickle the tracebacks.